manual/contents/src/operations/configuration-reference.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[configuration-settings]]
+= Configuration Settings Reference
+
+On this page you'll find the main configuration settings you can use with Neo4j.
+They can be set in <<file-locations, _neo4j.conf_>> (for configuring the embedded database see <<tutorials-java-embedded-setup-config>>).
```

Comment:
Could you remove this parenthesis completely? The embedded database will be handled in a separate document (Java developer's guide) in 3.0. However, skip it if it delays the PR. If so, we'll deal with it as part of the "rearrange docs" initiative.